commands/azureCommands/pull-from-azure.ts

```diff
@@ -71,7 +70,7 @@ async function pullImage(loginServer: string, imageRequest: string, username: st
                 // Temporary work-around for this error- same as Azure CLI
                 // See https://github.com/Azure/azure-cli/issues/4843
                 ext.outputChannel.show();
-                reject(new Error(`In order to log in to the Docker CLI using tokens, you currently need to go to \n${dockerConfigPath} and remove "credsStore": "wincred" from the config.json file, then try again. \nDoing this will disable wincred and cause Docker to store credentials directly in the .docker/config.json file. All registries that are currently logged in will be effectly logged out.`));
+                reject(new Error(`In order to log in to the Docker CLI using tokens, you currently need to go to \nOpen your Docker config file and remove "credsStore": "wincred" from the config.json file, then try again. \nDoing this will disable wincred and cause Docker to store credentials directly in the .docker/config.json file. All registries that are currently logged in will be effectly logged out.`));
```

Comment:
Why not point them directly to the config path like before?

Reply:
Because they can have an especial config, and that folder could not be in the default path.
So I didn't think it wasn't worth the trouble to look for the folder for a message that is there for a workaround that shouldn't be needed anymore. 
I was going to remove it all together but left it just as a precaution.